geoips/pydantic/feature_annotators.py

```diff
@@ -27,9 +28,31 @@ class CartopyFeature(PermissiveFrozenModel):
     )
     # NOTE: Once we add land / ocean features, we'll need to add another field, labeled
     # facecolor.
-    linewidth: float = Field(
+    linewidth: Optional[float] = Field(
         None, ge=0, description="The width in pixels of the specified feature."
     )
+    # TODO: There are more arguments here for a mpl Line2D artist and cartopy features.
+    # do we want to support all of those? Do we want to fail before or let
```

Comment:
I think let matplotlib fail! I don't think we need to duplicate their fail mechanisms!

Reply:
We're going to pull the matplotlib argument validation out into a separate issue and PR for further discussion. I definitely see your point about not reimplementing the matplotlib API, but it would be very nice if our documentation could actually show what arguments are available for a gridline annorator plugin.

It may be that this isn't possible, though, since we dont' really want to hard-code these arguments and matplotlib is kind of like a nesting doll when it comes to its artists and their arguments.